styles/mixins/button.js

```diff
@@ -1,15 +1,42 @@
+'use strict';
 module.exports = function () {
-  return function (mixin, bgColor, color) {
-    const activeBgColor = 'red';
-    const styles = {};
-    const state = {
-      'background-color': activeBgColor
-    };
-    styles['background-color'] = bgColor;
-    styles.color = color;
+  return function (mixin, bgColor, fgColor, type) {
+
+    const styles = {
+      display: 'inline-block',
+      'line-height': 1,
+      'text-align': 'center',
+      'font-size': '14px',
+      padding: '12px 14px',
```

Comment:
as of right now, we only have one size.  if we add another, we could add another param to the mixin